src/styles/utils/__variables.scss

```diff
@@ -0,0 +1 @@
+$transition: all 0.3s ease-in-out;
```

Comment:
do not use `all`
it is better to specify the elements to which you want to add a transition